terraso_backend/apps/core/migrations/0042_landscapedefaultgroup_group_enroll_method_and_more.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.2 on 2023-05-04 18:43
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("core", "0041_landscapedefaultgroup_alter_group_deleted_at_and_more"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="group",
+            name="enroll_method",
+            field=models.CharField(
+                choices=[("join", "Join"), ("invite", "Invite"), ("both", "Both")],
+                default="join",
+                max_length=10,
+            ),
+        ),
+        migrations.AlterField(
```

Comment:
Question: Also this one, were these migrations executed on staging? if that is the case then maybe we do need them.

Reply:
No I don't think so, I haven't deployed to staging yet.